src/main/java/mate/academy/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -36,10 +34,6 @@ public MovieSession add(MovieSession movieSession) {
                 transaction.rollback();
             }
             throw new DataProcessingException("Can't insert movie session" + movieSession, e);
-        } finally {
-            if (session != null) {
-                session.close();
-            }
```

Comment:
don't you need to close the session resources if we get some exception?
rollback these changes

Reply:
Same here. I've seen some moments I didn't like but decided not to fix the things I wasn't asked to.